lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8235,6 +8248,18 @@ static bool isUsingMacroName(clang::SourceManager &SM,
   return content == MacroName;
 }
 
+/// Utility function to import clang attributes from a source swift decl to synthesized swift decl.
```

Comment:
Minor: let's not duplicate the doc comment in the .cpp file. You've added a more elaborate comment in the header, which is great!